pkg/environment/api/project.go

```diff
@@ -252,10 +255,18 @@ func (s *EnvironmentService) CreateProject(
 	req *environmentproto.CreateProjectRequest,
 ) (*environmentproto.CreateProjectResponse, error) {
 	localizer := locale.NewLocalizer(ctx)
-	editor, err := s.checkSystemAdminRole(ctx, localizer)
+	editor, err := s.checkOrganizationRole(
+		ctx,
+		req.OrganizationId,
+		accountproto.AccountV2_Role_Organization_ADMIN,
+		localizer,
+	)
```

Comment:
I have fixed the issue where a regular admin user can't create or update a Project.